packages/expo-calendar/android/src/main/java/expo/modules/calendar/CalendarEvenBuilder.kt

```diff
@@ -6,10 +6,12 @@ import android.text.TextUtils
 import org.unimodules.core.arguments.ReadableArguments
 import java.util.*
 
-class CalendarEvenFactory(
+class CalendarEvenBuilder(
```

Comment:
isn't this a typo? 🤔 😅 
```suggestion
class CalendarEventBuilder(
```
it's also in the file name.